services/github/src/api.js

```diff
@@ -187,6 +187,7 @@ builder.declare({
   method: 'post',
   route: '/github',
   name: 'githubWebHookConsumer',
+  scopes: 'github:consume-webhook',
```

Comment:
This is a webhook called by GitHub, and is protected by its own access control (`X-Hub-Signature` or something like that) so I think `scopes: null` will make sense.